packages/tools/src/utilities/getCalibratedUnits.ts

```diff
@@ -173,7 +173,13 @@ const getCalibratedLengthUnitsAndScale = (image, handles) => {
     }
   } else if (calibration.scale) {
     scale = calibration.scale;
-    calibrationType = calibration?.type;
+  }
+
+  // everything except REGION/Uncalibratted
```

Comment:
REGION is excluded because it looks like there's handling above that adds it, and for uncalibrated, we don't display that one.